mysql-schema/027-userlog3.sql

```diff
@@ -0,0 +1,9 @@
+SET NAMES utf8mb4;
+START TRANSACTION;
+
+INSERT INTO changelog VALUES (27, UNIX_TIMESTAMP());
+
+ALTER TABLE user_log ADD
+  user_agent TEXT NOT NULL;
```

Comment:
This field should be nullable so previous entries already in the db don't fail

```suggestion
  user_agent TEXT NULL;
```